src/main/java/com/thindeck/steps/Nginx.java

```diff
@@ -136,7 +136,9 @@ public void update(@NotNull final String host, @NotNull final int hport,
      * @param sport Server port to redirect requests to
      * @return Commands for updating hosts configuration.
      */
-    private String updateConfigScript(final String host, final String server,
+    private String updateLoadBalancingScript(
```

Comment:
I think that right now we this method creates a script that updates load balancer, and it implements interface so both `update` and `LoadBalancing` are not required. So I would leave just `script(...)`